### PR TITLE
📝 Rework pricing doc

### LINKED ISSUE #2574: 📝 Rework pricing page
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/overview/pricing.html

### What exactly should be updated?

Make sure no information is there that doesn't need to be (as with the variables for names). Remove details with prices for individual services (how much something costs and so on). Details can be handled on marketing website: https://platform.sh/pricing/

At least for now, keep the page. Link to it from relevant other places (like the troubleshooting for disk storage).

Make it more clear who can edit plans (permissions).

Move to administration.

Update for:

- Content accuracy
- Includes what it should
- Doesn't include what doesn't fit
- Fits template
- Fits style guide

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2574

## What's changed

- Moved pricing to administration and moved order around a bit
- Reworked the page to be smoother and have no prices

### CODE REVIEW

File: docs/src/administration/pricing/_index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+## Agencies
+
+Agencies have access to several partner levels with many perks.
+
+* Agency partner levels: Registered, Bronze, Silver, Gold, Platinum, and Diamond
+* Free user licenses: Silver and above
+* Free development plan: Gold and above
+* Access to an agency-speciﬁc Small plan price: Bronze and above
+* Agency discounts for plans depending on the combination of the account tier and agency partner level:
```
Comment (trolologuy) [inline docs/src/administration/pricing/_index.md:118]: :question:
What's the `account tier` in that context?
Is this [the `Partner tier`](https://platform.sh/solutions/digital-agencies/) that changes with the credits?

:pick:
If there are no items for registered and Bronze, why not add `-` in the cells?

**Review by trolologuy — APPROVED** [review]
😻 The page looks much better now (and is clearer), while also fitting the style guide!
A few comments and ideas of improvement. Nothing blocking.

File: docs/src/administration/pricing/_index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+### Production environments
+
+Production environments are the live environments available to your users.
+Each Production plan has one Production environment that can be mapped to a [custom domain name](../../domains/steps/_index.md).
+
+The Production environment has more resources than the project's development environments.
+See the main [pricing page](https://platform.sh/pricing) for the resources available per plan for Production environments.
+
+### Development environments
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/administration/pricing/_index.md:40]: I think it's important to have plans and environments separately to set the correct expectations. The details for each are different, so I'd keep them separate. I also think grouping plans together and environments together is more useful.

I can reorder so at least the order is prod,dev plans, prod,dev environments.

File: docs/src/administration/pricing/_index.md (hunk 1 above)
Comment (CollierCZ) [reply docs/src/administration/pricing/_index.md:118]: The account tier is listed in the first column of the table.

File: docs/src/administration/pricing/_index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Development environments can't be mapped to [custom domain names](../../domains/steps/_index.md).
+Their resources are separate from Production environments.
+
+App containers on development environments default to a size of {{< partial "plans/default-dev-env-size" >}}.
+If you have the [`plans` permission](../organizations.md#manage-your-organization-users),
+but you can increase the size to match your production environment.
```
Comment (AnouckColson) [inline docs/src/administration/pricing/_index.md:47]:
```suggestion
you can increase the size to match your production environment.
```

File: docs/src/administration/pricing/_index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+You can add the following extras:
+
+* Users -- for an additional fee per user, except at certain [agency levels](#agencies).
+* Environments -- extra staging/development environments can be added in multiples of 3.
+* Storage -- extra storage can be added for a fee per 5&nbsp;GB per environment.
+  So if you have added 3 additional environments for a total of 6 development environments,
+  the fee is multiplied by 6.
+  
+  You can make any changes up to 100&nbsp;GB yourself.
+  For storage more than 100&nbsp;GB, create a [support ticket](https://console.platform.sh/-/users/~/tickets).
```
Comment (AnouckColson) [inline docs/src/administration/pricing/_index.md:84]:
```suggestion
  For storage over 100&nbsp;GB, create a [support ticket](https://console.platform.sh/-/users/~/tickets).
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]
😍 Very good, just noted a typo + a very small suggestion, otherwise I think it's ready to go!

**Review by AnouckColson — APPROVED** [review]
😍 All good!

**Pull request merged**